--- old
+++ new
@@ -49,2 +49,2 @@
-![Attachment-2024-01-13](/content/tools/playorna-cf-yxip/Attachment-2024-01-13.jpg)
-![Attachment-2024-01-13_1](/content/tools/playorna-cf-yxip/Attachment-2024-01-13_1.jpg)
+![Attachment-2024-01-13](Attachment-2024-01-13.jpg)
+![Attachment-2024-01-13_1](Attachment-2024-01-13_1.jpg)
@@ -52,1 +52,1 @@
-![Attachment-2024-01-13_2](/content/tools/playorna-cf-yxip/Attachment-2024-01-13_2.jpg)
+![Attachment-2024-01-13_2](Attachment-2024-01-13_2.jpg)