--- old
+++ new
@@ -69,1 +69,1 @@
-Orna 和 HoA 都是托管在 Cloudflare 的网络上，由于 Cloudflare CDN 的特点，它会提供大量的任播 ip ，每个 ip 只要你使用对应的 SNI ，都能连接到同一个网站。但你运营商不可能每次都给你分配延时最低和最快的，所以我们就需要优选一个 ip ，然后通过 adaway 来将游戏域名解析到这些 ip 上。
+Orna 和 HoA 都是托管在 Cloudflare 的网络上，由于 Cloudflare CDN 的特点，它会提供大量的任播 ip ，每个 ip 只要你使用对应的 SNI ，都能连接到同一个网站（不懂的可以搜索下，cloudflare 优选 ip ）。但你运营商不可能每次都给你分配延时最低和最快的，所以我们就需要优选一个 ip ，然后通过 adaway 来将游戏域名解析到这些 ip 上。