--- old
+++ new
@@ -2,1 +2,1 @@
-title: "缓解 Orna 在国内长翻书的问题"
+title: "解决 Orna/HoA 翻书的问题"
@@ -10,1 +10,1 @@
-# 缓解 Orna 在国内长翻书的问题
+# 解决 Orna/HoA 翻书的问题
@@ -12,0 +12,2 @@
+> 根据我长时间观察，这问题是全球玩家都会经常遇到的，ISP 与 CF CDN 连接的问题，详细原因可以查看[原理](#原理)。
+
@@ -75,0 +77,2 @@
+> ip 指向的是 icook.tw 的 cf cdn ，使用前先尝试打开 https://icook.tw
+
@@ -99,1 +103,1 @@
-Orna 和 HoA 都是托管在 Cloudflare 的网络上，由于 Cloudflare CDN 的特点，它会提供大量的任播 ip ，每个 ip 只要你使用对应的 SNI ，都能连接到同一个网站（不懂的可以搜索下，cloudflare 优选 ip ）。但你运营商不可能每次都给你分配延时最低和最快的，所以我们就需要优选一个 ip ，然后通过 adaway 来将游戏域名解析到这些 ip 上。
+Orna 和 HoA 都是托管在 Cloudflare 的 CDN 网络上，以提供相对更好的连接和缓存。由于 Cloudflare CDN 的特点，为每个网站分配几个cdn ip，根据 Cloudflare 的智能选择，会自动为用户提供网络运营商（ISP）到 Cloudflare CDN 节点的线路（不懂可以搜索下，cloudflare 优选 ip ）。然后问题就出现在这里， Cloudflare 也会根据时间调整线路，如果分配到较差的，就会卡顿。所以我们就需要优选一个 ip ，从而指定到相对较优的线路上避开，然后通过 adaway 来将游戏域名解析到这些 ip 上。
@@ -101,0 +105,6 @@
+```
+# 正常情况
+    ISP
+用户 =====>
+```
+